extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -138,7 +139,20 @@ export class AmpAdXOriginIframeHandler {
         this.adResponsePromise_,
         'timeout waiting for ad response').catch(e => {
           this.noContent_();
-          user().warn('AMP-AD', e);
+          getAdCid(this.baseInstance_).then(cid => {
+            const tagType = this.baseInstance_.element.getAttribute('type');
+            const TAG = `AMP-AD-${tagType}`;
```

Comment:
After searching through the repository, this seems to be fairly standard practice, if not convention. Wouldn't a class-level constant require that TAG always have a fixed value for all instances of amp-ad-xorigin-iframe-handler?